examples/ecr.tf

```diff
@@ -28,3 +30,40 @@ resource "kubernetes_secret" "example_team_ecr_credentials" {
   }
 }
 
+########SLACK NOTIFIATIONS OF ECR SCAN RESULTS########################
+
+#To send notifications to slack of the ECR scanned results, you may insert the following two modules, one creates the lambda function.
+# and the other creates the even bridge that forwards the result details to the lambda function
+
+# SLACK KUBERNETES SECRET 
+
+# Note that you will need to create a kubernetes secret that contains the slack token and the ECR repo.
+
+# The lambda needs the slack token and repo corresponding to the channel. The keys are set as environment variables. This secret needs
+# be created with the following keys:
+
+#Key 1: repo (without the pre-fix e.g if the url is 754256621582.dkr.ecr.eu-west-2.amazonaws.com/webops/webops-ecr1:rails, then in this case you need to supply 'webops/webops-ecr1')
+#Key 2: token
+
+# You can run the following kubernets example command to the create the secret: 
```

Comment:
I don't think we ever want teams to do this. Any secrets they want to create should *always* be in the form of git-crypted source code in git repos. Otherwise we're going to end up with secrets that get deleted and then nobody knows how to put them back (or even where they are) because the only ones who know have left.

Could you replace this to mention something like: "you can define the secret in a git-crypted yaml file like this ..."

Reply:
comments now reflect this method of creating the secert.